lib/shopify_import_massager.ex

```diff
@@ -24,35 +24,23 @@ defmodule ShopifyImportMassager do
 
     IO.puts("Processing START")
 
-    # We want the unchanged list of files so we can extract our substitutions
-    orders_input_files =
-      orders_files_filename_parameters
-      |> Map.merge(%{
-        input_path_prefix: input_folder,
-        output_path_prefix: "IGNORED",
-        number_padding_characters: 0
-      })
-      |> FilenameGenerator.generate_filename_pairs()
-      |> Enum.map(fn %{input_file: input_file} -> input_file end)
-
     substitution_map =
-      orders_input_files
-      |> Enum.map(fn file_name ->
-        SubstitutionMapExtractor.extract_substitution_map(file_name)
-      end)
-      |> List.flatten()
-
-    IO.puts("Order name substitutions extracted")
-
-    SubstitutionMapExtractor.show_substitutions(substitution_map, options)
+      extract_substitution_map(
+        %{
+          orders_files_filename_parameters: orders_files_filename_parameters,
+          input_folder: input_folder
+        },
+        options
+      )
 
     IO.puts("Processing order files with parameters #{inspect(orders_files_filename_parameters)}")
 
     %{
       substitution_map: substitution_map,
       input_folder: input_folder,
       output_folder: output_folder,
-      filename_parameters: orders_files_filename_parameters
+      filename_parameters: orders_files_filename_parameters,
+      file_type: :orders
```

Comment:
We now need the file type, since order name is in different columns for a specific file type